npm-debug.log

```diff
@@ -1,7 +1,7 @@
 0 info it worked if it ends with ok
```

Comment:
We should add the npm debug log to `.gitignore` it does not need to be online - it is a personal debugger.